arviz/data/io_cmdstan.py

```diff
@@ -372,15 +385,21 @@ def _read_output(path):
             timing_start = adaption_end + len(df) - warmup_rows
             timing_end = timing_start + timing_info_len
             # read timing_info
+            raise_timing_error = False
             for reading_line in range(timing_start, timing_end):
                 line = linecache.getline(path, reading_line)
                 if line.startswith("#"):
                     timing_info.append(line)
                 else:
-                    msg = "Invalid input file. " \
-                          "Header information missing from combined csv. " \
-                          "Timing: {}".format(path)
-                    raise ValueError(msg)
+                    raise_timing_error = True
+                    break
+            no_elapsed_time = not any("elapsed time" in row.lower() for row in timing_info)
```

Comment:
This is interesting. What was happening here that required this conditional?

Reply:
These are for combined csv `cat *.csv > comb.csv`

Assuming that time is missing it will either use break (end of the file) or after looping if it read header of the "second file"